### DISCUSSION TITLE
Arch AUR package

### DISCUSSION THREAD

**Discussion by ghost** [Xithrius/twitch-tui#213] [Ideas]
Hi, I created a package for AUR, but I'm too lazy to read guidelines and stuff in order to submit it, you or someone else may do that.

Also you can use this https://wiki.archlinux.org/title/VCS_package_guidelines#The_pkgver()_function

PKGBUILD:
```
# Maintainer: Name <youremail@domain.com>
pkgname=twitch-tui-git
pkgver=1.6.0
pkgrel=0
pkgdesc="Twitch chat in the terminal."
arch=('x86_64')
url="https://github.com/Xithrius/twitch-tui"
license=('MIT')
makedepends=('rustup>=1.25.1-1')
source=("https://github.com/Xithrius/twitch-tui/archive/refs/tags/v$pkgver.tar.gz")
md5sums=('47311fe780bfaac37184be20eefff37a')

prepare() {
	cd "twitch-tui-$pkgver"
  rustup default stable
}

build() {
	cd "twitch-tui-$pkgver"
	cargo build --release
}

check() {
	cd "twitch-tui-$pkgver"
  [ -r ./target/release/twt ]
  [ -x ./target/release/twt ]
}

package() {
	cd "twitch-tui-$pkgver"
	mkdir -p "${pkgdir}/usr/bin"
	mv ./target/release/twt "${pkgdir}/usr/bin"
}
```

**Comment by Xithrius** [comment]
Alright, I'll see what I can do.

**Comment by ghost** [comment]
Hi guys, didn't see this post, I already have two AUR packages, one for the stable release and the other one for the alpha
https://aur.archlinux.org/packages/twitch-tui
https://aur.archlinux.org/packages/twitch-tui-alpha

These are simple AUR packages, if you want own these packages let me know, or if you want to change the code or wathever.

**Reply by Xithrius** [reply]
The original person who created this post apparently deleted their account, interesting.

Thank you for creating these packages! I'll probably yoink them from you at a later time, once I figure out how the AUR works in terms of updating packages.
